backend/shared/rest/src/main/java/com/meemas/shared/Parser.java

```diff
@@ -0,0 +1,68 @@
+package com.meemas.shared;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class Parser {
```

Comment:
This name is way too generic -- what does it parse? Might be named `"RhsColon"`.